### PR TITLE
PPF-167 - Fix translated route name causing duplicate route names

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Changed
- [Use a language to prefix the route name](https://github.com/cultuurnet/publiq-platform/commit/9891ae8dc6344b359d5b554e0f8f9c6469effb2a)
- [Use getTranslatedRouteName for integrations redirect](https://github.com/cultuurnet/publiq-platform/commit/543285044e04b63f75b9d27bf1a7cb7d7f5f2c58)
- [Pass Accept-Language header to post request integrations](https://github.com/cultuurnet/publiq-platform/commit/6515fc02fdb0d7d4d19a1664b99a77764c4506d4)

### Fixed
- Fix translated route name causing duplicate route names

---
Ticket: https://jira.uitdatabank.be/browse/PPF-167

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -32,11 +32,11 @@
...
 Route::get('/auth/callback', Callback::class);
 
-TranslatedRoute::get(['/subscriptions', '/abonnementen'], [SubscriptionController::class, 'index']);
+TranslatedRoute::get(['en' => '/subscriptions', 'nl' => '/abonnementen'], [SubscriptionController::class, 'index']);
 
 Route::group(['middleware' => 'auth'], static function () {
-    TranslatedRoute::get(['/integrations', '/integraties'], [IntegrationController::class, 'index'], 'integrations.index');
-    TranslatedRoute::get(['/integrations/create', '/integraties/nieuw'], [IntegrationController::class, 'create']);
+    TranslatedRoute::get(['en' => '/integrations', 'nl' => '/integraties'], [IntegrationController::class, 'index'], 'integrations.index');
+    TranslatedRoute::get(['en' => '/integrations/create', 'nl' => '/integraties/nieuw'], [IntegrationController::class, 'create']);
```
Comment (LucWollants) [inline routes/web.php:39]: Maybe change the English one to `new` which matches better with `nieuw`

File: app/Domain/Integrations/Controllers/IntegrationController.php [hunk 2]
```diff
@@ -24,7 +25,7 @@ final class IntegrationController extends Controller
     public function __construct(
         private readonly SubscriptionRepository $subscriptionRepository,
         private readonly IntegrationRepository  $integrationRepository,
-        private readonly CurrentUser $currentUser
+        private readonly CurrentUser            $currentUser
```
Comment (LucWollants) [inline app/Domain/Integrations/Controllers/IntegrationController.php:28]: Some unwanted indentation.

File: app/Domain/Integrations/Controllers/IntegrationController.php [hunk 3]
```diff
@@ -87,6 +88,10 @@ public function store(StoreIntegration $storeIntegration): RedirectResponse
 
         $this->integrationRepository->save($integration);
 
-        return Redirect::route('integrations.index');
+        $language = $storeIntegration->headers->get('Accept-Language') ?? 'en';
```
Comment (LucWollants) [inline app/Domain/Integrations/Controllers/IntegrationController.php:91]: I see a mix of "default" languages. Here it is `en` and [here](https://github.com/cultuurnet/publiq-platform/pull/342/files#diff-a211f19734559dfaf8c7897fc66882ef1b6855b8ce9026e9226b43b796cd963dR33) it is `nl`.

Maybe add the default language as a method on `TranslatedRoute`?

**Review by LucWollants — APPROVED** [review]
Nice abstraction.

**Review by vhande — APPROVED** [review]

**Pull request merged**